### ISSUE TITLE
Embedded twitter posts are difficult to read in Firefox

### ISSUE THREAD

**Issue by mistachie777** [horst3180/Vertex-theme#65]
I'm running the latest version of the theme and I've noticed that embedded twitter feeds are almost impossible to read in Firefox because the text and background are both dark:

![twitter](https://cloud.githubusercontent.com/assets/9205048/6770624/1e3eef4c-d083-11e4-9a51-c2b35575495b.jpg)

I can't figure out what to add the userContent.css to fix this, but I'm assuming that's what needs to be adjusted.

**Comment by horst3180** [comment]
Hi, could you give me an example link? Does this happen exclusively with the dark theme?

**Comment by mistachie777** [comment]
I just checked this with the light theme and noticed that it did the same thing.  At that point I got suspicious that it wasn't because of the theme at all.  It turns out I had installed a Stylish theme for the blank pages before the userContent.css had been added to the theme and forgotten to disable it and that was somehow also affecting embedded twitter feeds.  Sorry about the false alarm!  However, in the process of testing this, I discovered another unrelated issue regarding tab theming, but I'll post that in another issue once I'm certain it's not because of anything else I've installed.  Thanks for quick reply.
